src/main/java/org/cryptomator/integrations/revealpaths/RevealPathsService.java

```diff
@@ -0,0 +1,32 @@
+package org.cryptomator.integrations.revealpaths;
+
+import org.cryptomator.integrations.common.IntegrationService;
+import org.cryptomator.integrations.common.IntegrationsLoader;
+
+import java.nio.file.NoSuchFileException;
+import java.nio.file.Path;
+import java.util.List;
+import java.util.stream.Stream;
+
+public interface RevealPathsService extends IntegrationService {
```

Comment:
I would rename it to `RevealPathService` (singular).

Reply:
What if the extend this interface to reveal more than one path?

Reply:
Changed in https://github.com/cryptomator/integrations-api/pull/16/commits/9d8080080d09324b0a2e29fadff6eda4b91295ad